addons/sale_stock/static/src/product_catalog/sale_order_line/sale_order_line.js

```diff
@@ -9,3 +9,15 @@ patch(ProductCatalogSOL, {
         deliveredQty: Number,
     },
 });
+
+patch(ProductCatalogSOL.prototype, {
+    get disableRemove() {
+        return this.props.quantity === this.props.deliveredQty;
+    },
+    get disabledButtonTooltip() {
+        if (this.disableRemove) {
+            return "The ordered quantity cannot be decreased below the amount already delivered. Instead, create a return in your inventory.";
```

Comment:
Need to translate that sentence. 🙂 

```suggestion
            return _t("The ordered quantity cannot be decreased below the amount already delivered. Instead, create a return in your inventory.");
```

Here is how to import the function : `import { _t } from "@web/core/l10n/translation";` 🙂

Reply:
yep obviously ^^' sry

Reply:
No problem. 🙂